tools/scylla-sstable.cc

```diff
@@ -188,6 +192,71 @@ class consumer_wrapper {
     }
 };
 
+class json_writer {
+    using stream = rapidjson::BasicOStreamWrapper<std::ostream>;
+    using writer = rapidjson::Writer<stream, rjson::encoding, rjson::encoding, rjson::allocator>;
+
+    stream _stream;
+    writer _writer;
+
+public:
+    json_writer() : _stream(std::cout), _writer(_stream)
+    { }
+
+    // following the rapidjson method names here
+    bool Null() { return _writer.Null(); }
```

Comment:
I want to avoid inheriting all the `String()` methods `_writer` has and instead want to provide my own overload. This is just cleaner and avoids any potential overload resolution ambiguity.

Reply:
Ok, I accept. Perhaps cleanest would have been to completely diverge from rapidjson's CamelCase function names and just call your functions by names you invent. But never mind, it's just a matter of style.